### ISSUE TITLE
openOrJoin and checkPresence function not working with SSEConnection  in php

### ISSUE THREAD

**Issue by KarenGrigoryan92** [muaz-khan/RTCMultiConnection#580]
Hi, you wrote that connection.socketURL is VERY_important but i don't have node.js, so i cant specify socketURL.
Is it openOrJoin SSEConnection ?
Please help me.
What i ought to do?

**Comment by muaz-khan** [comment]
Yes. `openOrJoin` can be added to [`dev/SSEConnection.js`](https://github.com/muaz-khan/RTCMultiConnection/blob/master/dev/SSEConnection.js) as well.

I'll add this method in the next commit, later today.

**Comment by KarenGrigoryan92** [comment]
When i using open and join functions separately everything working fine, but with openOrJoin connection cant be establish, i cant find out what i am doing wrong

**Comment by KarenGrigoryan92** [comment]
Also checkPresence function not calling callback function 
This is not working with SSEConnection 
connection.checkPresence(room_id, function(isRoomExist, roomid) {
            if (isRoomExist === true) {
                connection.join(roomid);
            } else {
                connection.open(roomid);
            }
        });

**Comment by muaz-khan** [comment]
Yes. `openOrJoin` relies on `checkPresence`. So both methods will be implemented in the next commit.

**Comment by KarenGrigoryan92** [comment]
Can you say please when are you going to commit sources?

**Comment by muaz-khan** [comment]
Please check this commit: https://github.com/muaz-khan/RTCMultiConnection/commit/b9bf7ff9ee6b25b56a6cab43acfe1447e20b4ab9

`checkPresence` is now supporting SSEConnection.

Added new file: [`demos/SSEConnection/checkPresence.php`](https://github.com/muaz-khan/RTCMultiConnection/blob/master/demos/SSEConnection/checkPresence.php)

**Comment by KarenGrigoryan92** [comment]
Sorry, but it does not working, i check you demo you added new button "Open or Join Room", your demo is not working to, i clicked that button, then i open new incognito window in chrome and  clicked "Open or Join Room" button and connection does not established.
checkPresence always return false

**Comment by bhavinshiroya1994** [comment]
screen sharing with audio not workig 

connection.session = {
audio:'two-way', //here i added
screen: true,
oneway: true
};
connection.sdpConstraints.mandatory = {
OfferToReceiveAudio:true, //here i change to true
OfferToReceiveVideo: false
};

connection.videosContainer = document.getElementById('videos-container');
connection.onstream = function(event) {
    connection.videosContainer.appendChild(event.mediaElement);
    event.mediaElement.play();
    setTimeout(function() {
        event.mediaElement.play();
    }, 5000);`
}

screen sharing work fine but audio not working please check this code what's wrong.

Thanks in advance
